docs/learn/dapp-developer/read-asset-data.md

```diff
@@ -265,9 +267,674 @@ console.log(assetMetadata);
+  '<myAssetAddress>',
+  lsp8Artifact.abi,
+  provider,
+);
+```
+
+  </TabItem>
+
+  <TabItem value="web3" label="web3">
+
+```bash
+npm install web3 @erc725/erc725.js @lukso/lsp-smart-contracts
+```
+
+```js
+// Add the necessary imports to your JS file
+import Web3 from 'web3';
+import lsp8Artifact from '@lukso/lsp-smart-contracts/artifacts/LSP8IdentifiableDigitalAsset.json';
+import { INTERFACE_IDS, ERC725YDataKeys } from '@lukso/lsp-smart-contracts';
+
+const web3 = new Web3(
+  new Web3.providers.HttpProvider('https://rpc.testnet.lukso.gateway.fm'),
+);
+// Create contract instance
+const myAssetContract = new web3.eth.Contract(
+  lsp8Artifact.abi,
+  '<myAssetAddress>',
+);
+```
+
+  </TabItem>
+
+</Tabs>
+
+:::caution
+
+Make sure to adjust `<myAssetAddress>` with the actual address of the asset. You can give it a try using a sample LSP8 asset on the LUKSO Testnet: [`0x8734600968c7e7193BB9B1b005677B4edBaDcD18`](https://wallet.universalprofile.cloud/asset/0x8734600968c7e7193BB9B1b005677B4edBaDcD18?network=testnet).
+
+:::
+
+### Preparing the Token IDs
+
+The [LSP8](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) allows for different [Token ID Formats](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat), meaning developers can specify their token IDs as:
+
+- `Number` of `uint256` (`TokenIdFormat`: `0` or `100`)
+- `String` with a maximum of 32 characters (`TokenIdFormat`: `1` or `101`)
+- `Smart Contract Address` with metadata (`TokenIdFormat`: `2` or `102`)
+- `Byte Identifier` with a maximum of 32 bytes (`TokenIdFormat`: `3` or `103`)
+- `Hash Digest` of 32 bytes (`TokenIdFormat`: `4` or `104`)
+
+To call the contract, you must first **prepare** your token IDs to match these standardized **Byte32 Hex Strings** based on the [`LSP8TokenIdFormat`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat) standardization. The **global token ID format** can be fetched from the [ERC725Y](../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) data storage using the [`getData()`](../../contracts/contracts/ERC725/ERC725.md#getdata) function:
+
+<Tabs groupId="provider-lib">
+  <TabItem value="ethers" label="ethers">
+
+```js
+// ...
+
+// Get the global token ID format
+let tokenIdFormat = parseInt(
+  await myAssetContract.getData(ERC725YDataKeys.LSP8['LSP8TokenIdFormat']),
+  16,
+);
+
+console.log(tokenIdFormat);
+// 0
+```
+
+  </TabItem>
+
+  <TabItem value="web3" label="web3">
+
+```js
+// ...
+
+// Get the global token ID format
+let tokenIdFormat = parseInt(
+  await myAssetContract.methods
+    .getData(ERC725YDataKeys.LSP8['LSP8TokenIdFormat'])
+    .call(),
+  16,
+);
+
+console.log(tokenIdFormat);
+// 0
+```
+
+  </TabItem>
+
+</Tabs>
+
+Its value can be interpreted as the following:
+
+- `0`, `1`, `2`, `4` means the token ID format is **equal across all token IDs**
```

Comment:
Again same thing, this is already mentioned in the Standard page. This page should be a How To guide, not a page to explain the standard again.